src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +12,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        if (UserService.findByEmail(email) != null) {
```

Comment:
I would strongly recommend to make `UserService` a class member. It will improve flexibility and simplify testing. Also consider fetching user by email once at the beginning, hence we will not need to call `findByEmail` method each time we need to conduct some checks.